frontend/src/app/components/LocalizedHtml.js

```diff
@@ -0,0 +1,14 @@
+
+import { Localized } from 'fluent-react/compat';
+import parser from 'html-react-parser';
+import React from 'react';
+
+export default class LocalizedHtml extends Localized {
```

Comment:
Yes, it is used in ExperimentPage. Github "helpfully" hides the diff because it is too large, so searching for <LocalizedHtml doesn't show anything :-)